EKF/ekf.cpp

```diff
@@ -139,9 +141,17 @@ bool Ekf::initialiseFilter()
 {
 	// Keep accumulating measurements until we have a minimum of 10 samples for the required sensors
 
-	// Sum the IMU delta velocity measurements
+	// Filter accel for tilt initialization
 	const imuSample &imu_init = _imu_buffer.get_newest();
-	_delVel_sum += imu_init.delta_vel;
+	// Reset to imu sample on boot up
+	// Use mag_counter to reduce number of variables
+	if(_mag_counter == 0){
```

Comment:
Not entirely happy about this reuse of mag_counter

Reply:
Me neither, do you see any other solution?

Reply:
Just use a boolean with a meaningful name that gets set to true only once.